apps/daimo-mobile/src/action/useSendAsync.ts

```diff
@@ -123,6 +128,27 @@ export function transferAccountTransform(namedAccounts: EAccount[]) {
   };
 }
 
+function authenticatedDataAccountTransform(
```

Comment:
the name is not super obvious what it does. maybe something like `addInviteLinkStatus`?

i'm not quite sure what you mean by "authenticated data"

Reply:
renamed -- I meant in the sense of `AD` as used in cryptography lol, data that has been revealed because the the account was authenticated, added a comment also.